test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,61 @@
+const agent = require('superagent-promise')(require('superagent'), Promise);
+const statusCode = require('http-status-codes');
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+
+chai.use(chaiSubset);
+const { expect } = chai;
+const aperdomoGithubUrl = 'https://api.github.com/users/aperdomob';
+const allReposPath = '/repos';
+const repoContentsUrl = 'https://api.github.com/repos/aperdomob/jasmine-awesome-report/contents/';
+// const url = 'https://api.github.com/repos/aperdomob/jasmine-awesome-report/contents';
+
+describe('Consuming get services from github API', () => {
+  it('Verify name, company and location', () => {
+    const query = {
```

Comment:
declara las constantes fuera del it, el it sólo se utiliza para verificar con los expect

Reply:
puedes usar before para este fin